frontend/src/components/UserSearch.vue

```diff
@@ -132,7 +132,19 @@
           :loading="userSearchLoadingStatus"
           loading-text="Searching for users"
           v-on:click:row="selectUser"
-        ></v-data-table>
+        >
+          <template v-if="searchResults.length > 0" v-slot:footer>
+            <v-toolbar flat>
+              <v-spacer/>
```

Comment:
This template, toolbar, spacer stuff is just for a bit of layout. The `download-csv` element is all that matters.